Modules/Core/Common/src/itkMultiThreaderBase.cxx

```diff
@@ -535,11 +535,6 @@ MultiThreaderBase::ParallelizeImageRegion(unsigned int
   }
   ProgressReporter progress(filter, 0, 1);
 
-  SizeValueType pixelCount = 1;
-  for (unsigned d = 0; d < dimension; ++d)
-  {
-    pixelCount *= size[d];
-  }
   struct RegionAndCallback rnc
   {
     funcP, dimension, index, size, 0, filter
```

Comment:
It appears that `pixelCount` was originally passed as initializer value to the `RegionAndCallback` object. Now it is just zero (`0`). With "Progress reporting revisionv5" https://github.com/InsightSoftwareConsortium/ITK/pull/1575 commit https://github.com/InsightSoftwareConsortium/ITK/pull/1575/commits/c8787b8ed8696d8ee6b0ddef0a377695921392f2 3 March 2020

If you think `pixelCount` may be removed here, I guess it should also be removed from the definition of `RegionAndCallback`, right?

https://github.com/InsightSoftwareConsortium/ITK/blob/v5.2.0/Modules/Core/Common/include/itkMultiThreaderBase.h#L444

Reply:
@dzenanz @blowekamp what is the intended approach here?

Reply:
Hrmm... Just glancing through #1575 ( that is a big PR), I think a comment should have been left here as to why it is this way. It may be unfortunately used as a "magic" value to distinguished one reporting progress method vs another. Looking into how `RegionaAndCallback::pixelCount` is use ( or not used ) should clarify the purpose of the zero value or indicate it's not needed.

Either way the code is not used.

Reply:
Was the calculated `pixelCount` meant to be passed to RegionAndCallback, or not?

Reply:
> Was the calculated `pixelCount` meant to be passed to RegionAndCallback, or not?

The current zero value is correct and working as far as I know. As I said if may be a "magic" value. 

The `pixelCount` computation can be removed.